.github/workflows/tests.yml

```diff
@@ -35,11 +35,11 @@ jobs:
         with:
           python-version: 3.6
           cache: "pip"
-          cache-dependency-path: "requirements-test.txt"
```

Comment:
Is requirements-test.txt cached by default?

Reply:
Oh, right, forgot that I changed the file name. Reverting that 👍